### ISSUE TITLE
System breaks IDT export line if it is longer than 80 chars

### ISSUE THREAD

**Issue by AlexeyGirin** [epam/Indigo#1986]
**Steps to Reproduce**
1. Switch to Macro
2. Load from file: [System breaks IDT export line if it is longer than 80 chars.zip](https://github.com/user-attachments/files/15592884/System.breaks.IDT.export.line.if.it.is.longer.than.80.chars.zip)
OR create on the canvas followng configuration:
![image](https://github.com/epam/Indigo/assets/26869421/f08ff4f4-17f9-4653-81b7-a15b207ff21e)
3. Try to export canvas to IDT

**Actual behavior**
System show the following result:
```
/52MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA
//i2MOErA//i2MOErA//i2MOErA//3Phos/
```
![image](https://github.com/epam/Indigo/assets/26869421/834ac258-b931-4bdd-895d-672209df6e41)

**Expected behavior**
System show the following result (solid line, w/o new line breaks):
`/52MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//3Phos/`

**Versions:**
 - Indigo Toolkit Version 1.21.0-rc.1.0-g9194599b2-wasm32-wasm-clang-19.0.0
 - Ketcher Version 2.22.0-rc.2 Build at 2024-06-03; 18:43:00
 - Chrome Version 125.0.6422.142 (Official Build) (64-bit)
 - Windows 10

Issue found while testing https://github.com/epam/Indigo/issues/1900

**Comment by AnnaKosol** [comment]
**Actual result:**
System show the result with new line breaks
<img width="1512" alt="Снимок экрана 2024-07-03 в 12 44 41" src="https://github.com/epam/Indigo/assets/174325443/6ed2bb09-e95e-4e62-b100-2b11fd22a221">

**Expected result:**
System show the following result (solid line, w/o new line breaks):
/52MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//i2MOErA//3Phos/

Versions:
Ketcher Version 2.23.0-rc.1 Build at 2024-07-02; 09:03:38
Indigo Version 1.21.0-rc.1.0-g9194599b2-x86_64-linux-gnu-11.2.1
MacOs Sonoma version 14.5
Chrome Version 126.0.6478.127 (Официальная сборка), (arm64)

**Comment by AliaksandrDziarkach** [comment]
It was tested in Indigo Version 1.21.0-rc.1.0-g9194599b2-x86_64-linux-gnu-11.2.1 but fix in the Indigo 1.22-rc.1

**Comment by AlexeyGirin** [comment]
Fixed.
![image](https://github.com/user-attachments/assets/9effa412-065e-4759-951d-ce78017c7142)

- Ketcher Version 2.23.0-rc.2 Build at 2024-07-10; 19:39:22
- Indigo Toolkit Version 1.22.0-rc.2.0-ga21fb5d8f-wasm32-wasm-clang-19.0.0
- Windows 10 Pro
- Chrome Version 126.0.6478.127 (Official Build) (64-bit)
